example/spec/loadTest/runScaledTest.js

```diff
@@ -0,0 +1,362 @@
+/* eslint-disable no-await-in-loop */
+
+'use strict';
+
+const path = require('path');
+const fs = require('fs');
+const pWaitFor = require('p-wait-for');
+
+const { getAggregateMetricQuery, getInvocationCount } = require('@cumulus/integration-tests/metrics');
+const { generateIterableTestDirectories } = require('@cumulus/integration-tests/utils');
+const { LambdaStep } = require('@cumulus/integration-tests/sfnStep');
+const { listS3ObjectsV2 } = require('@cumulus/aws-client/S3');
+const { providers, collections, granules } = require('@cumulus/api-client');
+const { randomString } = require('@cumulus/common/test-utils');
+
+const {
+  buildAndExecuteWorkflow,
+  api: apiTestUtils,
+} = require('@cumulus/integration-tests');
+
+const {
+  createTestSuffix,
+  createTimestampedTestId,
+  loadConfig,
+} = require('../helpers/testUtils');
+
+// ** Configurable Variables
+const expectedGranuleCount = 500; // Per batch
+const batches = 5; // Number of batches to run
+const providerPathTemplate = `ingest_${expectedGranuleCount}_test`; // Directory pattern to use
+const providerPaths = generateIterableTestDirectories(providerPathTemplate, batches); // Given N batches, run over "pattern_0-9,a-z" directories
+// const providerPaths = ['ingest_100_test_0'];
```

Comment:
Leftover comment from testing?

Reply:
This was deliberate, but could be tidied up.

Reply:
227f01512